module/bdev/nvme/bdev_nvme.c

```diff
@@ -200,6 +200,7 @@ struct spdk_thread *g_bdev_nvme_init_thread;
 static struct spdk_poller *g_hotplug_poller;
 static struct spdk_poller *g_hotplug_probe_poller;
 static struct spdk_nvme_probe_ctx *g_hotplug_probe_ctx;
+static enum spdk_nvme_transport_type g_nvme_trtype = SPDK_NVME_TRANSPORT_CUSTOM;
```

Comment:
Why do we choose `SPDK_NVME_TRANSPORT_CUSTOM` rather than `SPDK_NVME_TRANSPORT_TCP`?

Reply:
We are still using `trid->trtype` (TCP in our case) as the default. Initializing it with `SPDK_NVME_TRANSPORT_CUSTOM` is just a placeholder to indicate that the transport type is initially unset.
https://github.com/c3y1huang/longhorn-spdk/blob/9834-v2-interrupt-mode-v25.05/module/bdev/nvme/bdev_nvme.c#L6743-L6744